### DISCUSSION TITLE
How to open a single file with electron build ?

### DISCUSSION THREAD

**Discussion by bayang** [eclipse-theia/theia#10128] [General] [answered]
Hi,
I built an electron version of Theia on linux.

If I try to open a file (like I do with other editors) I have a workspace error.

eg from CLI : 
`/home/me/apps/Theia-1.2.0_bbf0f8c08a3ed484b943fbae7c05c8e1.AppImage README.md`
I have this error in the logs :
`root WARN Not a valid workspace file: /home/me/media/theia-apps/README.md`

Same from the GUI, in my file explorer it is not possible to right click a file > open with > theia

I think an editor should allow this.

**Comment by msujew** [comment, marked as answer]
Hi @bayang, thank you for your question.

This is a quite old, known issue. Theia currently does not allow opening files independently of a workspace, see #3354 for more info.

**Reply by bayang** [reply]
Ok thanks !
